redash/controllers.py

```diff
@@ -80,8 +84,8 @@ def login():
                            analytics=settings.ANALYTICS,
                            next=request.args.get('next'),
                            username=request.form.get('username', ''),
-                           show_google_openid=settings.GOOGLE_OAUTH_ENABLED)
-
+                           show_google_openid=settings.GOOGLE_OAUTH_ENABLED,
+                           show_saml_login=settings.SAML_LOGIN_ENABLED)
```

Comment:
You set this flag, but I don't see you're using it. Not implemented yet, or just a missing file in the pull request?